src/components/Form/TextInput.tsx

```diff
@@ -20,7 +20,7 @@ const inputBaseClassName = `${baseClassName}_input`;
 
 export interface TextInputLabelProps extends HtmlLabelProps {}
 
-type Label = 'default' | 'screenreader';
+type Label = 'hidden' | 'visible';
```

Comment:
Hmm.. would it possible to use enum instead? 

E.g:
`
enum Label {
  Hidden = "hidden",
  Visible = 'visible'
};
`

Then we might avoid possible typo when later using enum values instead of strings.

`const hideLabel = labelMode === Label.Hidden;`